config/basic/montagu.yml

```diff
@@ -32,8 +32,36 @@ api:
 db:
   name: montagu-db
   tag: master
-  user: vimc
-  password: changeme
+  root_user: vimc
+  migrate:
+    name: montagu-migrate
+    tag: master
+  users:
+    api:
+      password: "apipassword"
```

Comment:
these will all come from vault; I'll add a vault config and some tests in a further PR